tests/ui/resolve/issue-120559.rs

```diff
@@ -0,0 +1,5 @@
+use std::io::Read;
+
+fn f<T: Read, U, Read>() {} //~ ERROR expected trait, found type parameter `Read`
```

Comment:
We could suggest explicitly qualifying `Read` here if feasible: `T: std::io::Read` or even `self::Read` to pick up the import item. cc https://github.com/rust-lang/rust/issues/109950#issuecomment-1496787799